src/utils/constants.js

```diff
@@ -52,6 +52,7 @@ export const DIALOG_ACTION_INITIAL_STATE = {
   item: null,
   open: false,
   destructive: false,
+  mode: '',
```

Comment:
GIven that `mode` won't be inputted to a text field that expects a string, we could probably just make this `null`.